essay_on_tigers.txt

```diff
@@ -0,0 +1,7 @@
+Tigers are magnificent creatures that belong to the cat family. They are known for their distinct orange fur with black stripes, which act as camouflage in their natural habitat. Tigers are the largest of all big cats and are renowned for their strength and agility.
+
```

Comment:
Consider breaking this long sentence into multiple shorter sentences for better readability.